openbolt.gemspec

```diff
@@ -6,14 +6,14 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 require 'bolt/version'
 
 Gem::Specification.new do |spec|
-  spec.name          = "bolt"
+  spec.name          = "openbolt"
   spec.version       = Bolt::VERSION
```

Comment:
This wasn’t working for openfact version bump from the release prep, let’s see if we can get it working here because DRY